packages/docs/docs/experimental/budget-automation.md

```
@@ -0,0 +1,356 @@
+- And much more!
+
+## Notes Templates & UI Migration
+For any given category, there is one source of truth for Budget automations, either the UI or the notes, not both.
+
+If you have a notes template, the first time you open the UI (clock icon) you will be presented with migration automation(s). Click to show the original notes templates at the top of the form for comparison.
+
+![alt-text-here](/img/goal-template/migration.webp)
+
+Make any changes you feel are needed, then _Save_ the form to complete the migration. The notes can now be deleted. 
+
+## Creating Automations {/* #create-automations */}
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)